src/Type/WPInterfaceType.php

```diff
@@ -17,24 +17,41 @@ class WPInterfaceType extends InterfaceType {
 	public $type_registry;
 
 	/**
-	 * @var array<string,mixed>
+	 * @var array{
+	 *     name?: string|null,
+	 *     description?: string|null,
+	 *     fields: (callable(): iterable<string, mixed>)|iterable<string, mixed>,
+	 *     interfaces?: (callable(): iterable<callable(): \GraphQL\Type\Definition\InterfaceType|\GraphQL\Type\Definition\InterfaceType>)|iterable<callable(): \GraphQL\Type\Definition\InterfaceType|\GraphQL\Type\Definition\InterfaceType>,
+	 *     resolveType?: (callable(mixed, mixed, \GraphQL\Type\Definition\ResolveInfo): (callable(): (\GraphQL\Type\Definition\ObjectType|string|null)|\GraphQL\Deferred|\GraphQL\Type\Definition\ObjectType|string|null))|null,
```

Comment:
Where did these type definitions come from? I'm trying to understand whether  it's the types that are incorrect or if our code is out of date with what is now expected with WPGraphQL v15

E.g 
![image](https://github.com/user-attachments/assets/7631c734-7a6f-4e9d-91b2-985b5862ac7d)

Reply:
https://github.com/wp-graphql/wp-graphql/actions/runs/13102963390/job/36553507641